mobile/src/libs/components/button/button.tsx

```diff
@@ -9,24 +9,34 @@ type Properties = {
   label: string;
   onPress: () => void;
   isDisabled?: boolean;
+  style?: 'solid' | 'outlined';
 };
 
 const Button: React.FC<Properties> = ({
   label,
   onPress,
   isDisabled = false,
+  style = 'solid',
 }) => {
   const handleOnPress = (): void => {
     onPress();
   };
 
   return (
     <Pressable
-      style={[styles.button, isDisabled && styles.buttonDisabled]}
+      style={[
+        style === 'solid' ? styles.button : styles.buttonSurvey,
```

Comment:
Let's reduce the dependency of a goal for button, as example check thread-js please

Reply:
@v0ldemar01 i quite dont understand how should it be, i checked thread-js and i saw this for button, and mine code looks almost the same and it works properly, because i tried different combinations and didnt get required effects, only with ,y code i got behavior that i wanted
![Снимок экрана 2023-08-31 131754](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/111873795/525114d1-3d65-4bad-a5c9-05998de6c2d0)